apps/hash-frontend/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "description": "HASH frontend",
   "license": "AGPL-3.0",
+  "type": "module",
```

Comment:
Adding this also requires renaming `next.config.js` and `buildstamp.js` to `.cjs` files, otherwise the frontend won't run (via `yarn dev`)... but even then when running the frontend I get a `404` for the homepage. If I remove `"type": "module"` it appears to work. Do you remember what error forced this?

Reply:
That's an interesting one. I'll take a look at it again.